lib/permutat.g

```diff
@@ -444,39 +444,26 @@ SetOne( PermutationsFamily, () );
 
 #############################################################################
 ##
-#F  ListPerm( <perm>[, <length>] )  . . . . . . . . . . . . .  list of images
+#F  ListPerm( <perm>[, <n>] )  . . . . . . . . . . . . .  list of images
 ##
 ##  <#GAPDoc Label="ListPerm">
 ##  <ManSection>
-##  <Func Name="ListPerm" Arg='perm[, length]'/>
+##  <Func Name="ListPerm" Arg='perm[, n]'/>
 ##
 ##  <Description>
 ##  is a list <M>l</M> that contains the images of the positive integers
-##  under the permutation <A>perm</A>.
+##  from 1 to <A>n</A> under the permutation <A>perm</A>.
 ##  That means that
 ##  <M>l</M><C>[</C><M>i</M><C>]</C> <M>= i</M><C>^</C><A>perm</A>,
-##  where <M>i</M> lies between 1
-##  and the largest point moved by <A>perm</A>
-##  (see&nbsp;<Ref Attr="LargestMovedPoint" Label="for a permutation"/>).
+##  where <M>i</M> lies between 1 and <A>n</A>.
 ##  <P/>
-##  An optional second argument specifies the length of the desired list.
+##  If the optional second argument <A>n</A> is omitted then the largest
+##  point moved by <A>perm</A> is used
```

Comment:
The implemented function doesn't use the largest moved point of the perm, it seems that the current version in GAP 4.12.2 (at least) does use the largest moved point. I'll comment again in the right place.